src/sbt-test/sbt-bundle/bundleConfig/build.sbt

```diff
@@ -0,0 +1,42 @@
+import ByteConversions._
+import com.typesafe.sbt.bundle.SbtBundle._
+import org.scalatest.Matchers._
+
+lazy val root = (project in file(".")).enablePlugins(JavaAppPackaging)
+
+name := "simple-test"
+
+version := "0.1.0-SNAPSHOT"
+
+BundleKeys.nrOfCpus := 1.0
+BundleKeys.memory := 64.MiB
+BundleKeys.diskSpace := 10.MB
+BundleKeys.roles := Set("web-server")
+
+BundleKeys.endpoints += "other" -> Endpoint("http", 0, Set(URI("http://:9001/simple-test")))
+BundleKeys.endpoints += "akka-remote" -> Endpoint("tcp")
```

Comment:
removing the BundleKeys results in sbt.Init stack trace:

    [info] References to undefined settings: 
[info] 
[info]   *:bundleMemory from *:bundleStartCommand ((com.typesafe.sbt.bundle.SbtBundle) SbtBundle.scala:144)
[info] 
[info]   *:bundleMemory from *:bundleStartCommand ((com.typesafe.sbt.bundle.SbtBundle) SbtBundle.scala:144)
[info] 
[info]   *:bundleDiskSpace from *:bundleConf ((com.typesafe.sbt.bundle.SbtBundle) SbtBundle.scala:142)
[info] 
[info]   *:bundleMemory from *:bundleConf ((com.typesafe.sbt.bundle.SbtBundle) SbtBundle.scala:142)
[info] 
[info]   *:bundleNrOfCpus from *:bundleConf ((com.typesafe.sbt.bundle.SbtBundle) SbtBundle.scala:142)
[info]